docs/MVS_Outputs.rst

```diff
@@ -62,6 +62,15 @@ The LCOElectricity are then calculated with:
 
 ** The content of this section was copied from the conference paper handed in to CIRED 2020**
 
+Levelized Cost of Energy of Asset (LCOE_ASSET)
+################################
+
+This KPI measures the cost of generating 1 kWh for each asset in the system. It can be used to assess and compare the available alternative methods of energy production. The levelized cost of energy of an asset (LCOE_ASSET) is usually obtained looking at the lifetime costs of building and operating the asset per unit of total energy generated by the asset over the assumed lifetime [currency/kWh].  
```

Comment:
```suggestion
This KPI measures the cost of generating 1 kWh for each asset in the system. It can be used to assess and compare the available alternative methods of energy production. The levelized cost of energy of an asset (LCOE_ASSET) is usually obtained looking at the lifetime costs of building and operating the asset per unit of total energy throughput of an asset over the assumed lifetime [currency/kWh].  
```